src/openms/source/ANALYSIS/QUANTITATION/TMTEighteenPlexQuantitationMethod.cpp

```diff
@@ -0,0 +1,219 @@
+    defaults_.setValue("channel_131N_description", "", "Description for the content of the 131N channel.");
+    defaults_.setValue("channel_131C_description", "", "Description for the content of the 131C channel.");
+    defaults_.setValue("channel_132N_description", "", "Description for the content of the 132N channel.");
+    defaults_.setValue("channel_132C_description", "", "Description for the content of the 132C channel.");
+    defaults_.setValue("channel_133N_description", "", "Description for the content of the 133N channel.");
+    defaults_.setValue("channel_133C_description", "", "Description for the content of the 133C channel.");
+    defaults_.setValue("channel_134N_description", "", "Description for the content of the 134N channel.");
+    defaults_.setValue("channel_134C_description", "", "Description for the content of the 134C channel.");
+    defaults_.setValue("channel_135N_description", "", "Description for the content of the 135N channel.");
+
+    defaults_.setValue("reference_channel", "126", "The reference channel (126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131N, 131C, 132N, 132C, 133N, 133C, 134N, 134C, 135N).");
+    defaults_.setValidStrings("reference_channel", TMTEighteenPlexQuantitationMethod::channel_names_);
+
+    // TODO: verify these
+    defaults_.setValue("correction_matrix", std::vector<std::string>{"0.0/0.0/9.40/0.34",
+                                                                      "0.0/0.78/9.41/0.33",
+                                                                      "0.0/0.93/8.98/0.28",
+                                                                      "0.0/1.47/8.13/0.26",
+                                                                      "0.0/1.47/7.25/0.15",
+                                                                      "0.0/2.74/6.86/0.15",
+                                                                      "0.13/2.59/6.39/0.10",
+                                                                      "0.13/2.68/5.58/0.10",
+                                                                      "0.04/3.10/5.24/0.08",
+                                                                      "0.03/3.41/4.57/0.12",
+                                                                      "0.08/3.90/4.04/0.04",
+                                                                      "0.16/4.30/1.80/0.0",
+                                                                      "0.11/4.55/2.29/0.0",
+                                                                      "0.08/3.87/3.49/0.03",
+                                                                      "0.22/4.96/1.37/0.0",
+                                                                      "0.33/6.11/1.14/0.0",
+                                                                      "0.14/5.81/0.31/0.0",
+                                                                      "0.21/5.42/0.00/0.0"},
```

Comment:
not sure about the defaults there.
If we take the example sheet from above, we should only use the number from a single cell (the one which is mapped) and not add up both values from the sub-columns. The contribution from the channel which is currently not mapped should be independent of the mapped channel, hence our error is smaller if we do not sum up the two values .right?

Reply:
Yes I realized that too. I was confused by the +1Da +2Da headers of the columns and went against my own suggestion of just using the C13 values.